src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -216,11 +222,31 @@ export default function SelectNextPage() {
 
           <Library.Example title="Disabled Select">
             <Library.Demo title="Disabled Select">
-              <div className="w-[350px] mx-auto">
+              <div className="w-96 mx-auto">
                 <SelectExample disabled />
               </div>
             </Library.Demo>
           </Library.Example>
+
+          <Library.Example title="Select with long content">
+            <p>
+              <code>SelectNext</code> makes sure the content never overflows,
+              but you may need to tweak <code>buttonContent</code> or options
+              content in cases where more complex content needs to be rendered.
+            </p>
+
+            <Library.Demo title="Plain text">
+              <div className="mx-auto">
```

Comment:
The width and left position of the select control changes depending on the selected item. That looks weird to me, and it means that the dropdown arrow moves around. I think the widths of these controls is almost always fixed.

Reply:
> The width and left position of the select control changes depending on the selected item

This is precisely what I was trying to prevent. I think I may have broken in some of my latest changes before pushing the branch.

It should behave like in the second screenshot I added to the PR description. I'll try to figure it out.

Reply:
Yeah, I see the problem.

In order for this to work, the dimensions need to be somehow constrained, either by the `SelectNext` itself (like here) or its container. Otherwise, it can potentially grow.

In this case, since I'm trying to constrain it through `SelectNext`'s own `classes` prop, it needs to be `!w-36`, not `w-36`, otherwise the `w-full` it already has will take precedence.

Another option would be to constrain it through the container (I think that's what I did initially, when I took the screenshots), but it's important the other option also works, in case we need to use it inside an `InputGroup`.

Having to add the `important` modifier is not ideal (it means you need to know the internal details), but we can think on a better way to address that later.